utils.js

```diff
@@ -398,8 +398,8 @@ function isDomElement(thing) {
  * execute any of its inline scripts.
  */
 function staticDom(html) {
-    const {jsdom} = require('jsdom');
-    return jsdom(html, {features: {ProcessExternalResources: false}});
+    var parser = new DOMParser();
```

Comment:
Ooo, a handy thing! So, I see the `<script>` are still in there. This makes me suspect our test harness actually fetches those with remote sources at runtime. For speed, durability, and the ability to work offline, test harnesses shouldn't hit the network. Is there something we can do about that?